### PR TITLE
[HotFix] Fix unpackdb after merging #2800

### PR DESCRIPTION
**Pull request by apwojcik**
PR #2800 broke the configuration of a CMake build on Windows.

### CODE REVIEW

**Comment by junliume** [comment]
Likely #2803 will not be necessary if this PR is merged. However, I am testing performance impact of this PR (the purpose of #2800 is to fix a performance related issue)

**Review by junliume — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -531,26 +531,25 @@ function(unpack_db db_bzip2_file suffix_var)
...
         )
         add_custom_target(generate_${__tname}_txt ALL DEPENDS ${KERNELS_BINARY_DIR}/${__fname}.txt)
         add_dependencies(generate_kernels generate_${__tname}_txt)
-        set("${suffix_var}" ".txt" PARENT_SCOPE)
+        set(__fname ${__fname}.txt)
     else()
         add_dependencies(generate_kernels generate_${__tname})
-        set("${suffix_var}" "" PARENT_SCOPE)
     endif()
+    set(__fname ${__fname} PARENT_SCOPE)
```
Comment (atamazov) [inline CMakeLists.txt:543]: @apwojcik [Recommendation] Rename `__fname` to `unpack_db_output_fname` or something alike to emphasize that value of this variable is produced by unpack_db.

**Review by atamazov — APPROVED** [review]
LGTM!

File: CMakeLists.txt (hunk 1 above)
Comment (apwojcik) [reply CMakeLists.txt:543]: @atamazov That makes sense. This PR is already merged. I will prepare a separate PR to change the return value's name.

**Pull request merged**